get-started/first-steps-vs.md

```diff
@@ -36,23 +34,14 @@ To start the wizard, use the [Project](#project-menu) menu.
 
     ![Create a new project dialog with maui in the search field and results](../installation/vs-integration/images/vsextensions_createapp.png)
 
-1. Select one of the supported project templates:
-
-     * **Telerik .NET MAUI App - Configurable Wizard**&mdash;Represents a configurable wizard, which contains various predefined application templates that include the Telerik .NET MAUI suite setup. 
-     * **Telerik .NET MAUI Blank App**&mdash;Represents a blank project for creating a .NET MAUI application that includes the Telerik .NET MAUI suite setup.
-     * **Telerik .NET MAUI Blank Shell App**&mdash;Creates a blank .NET MAUI Shell application. You can use this project as a .NET MAUI Shell example. In addition, you can use features like Shell Navigation, .NET MAUI Shell flyout, and .NET MAUI Shell tabs. For more details on .NET MAUI Shell, review the official [Microsoft documentation](https://learn.microsoft.com/en-us/dotnet/maui/fundamentals/shell/). 
-
+1. Select the **Telerik .NET MAUI Blank App** which represents a blank project for creating a .NET MAUI application that includes the Telerik .NET MAUI suite setup.
 
 ## Step 3: Configure the Project
 
-1. The next step lets you configure your **Telerik UI for .NET MAUI** project by setting your project name and its location.
+Configure your **Telerik UI for .NET MAUI** project by setting your project name and its location.
 
     ![Telerik .NET MAUI App configurable wizard initial screen within the Create your new project dialog](../installation/vs-integration/images/vsextensions_configureapp.png)
```

Comment:
Please double-check if the image gets rendered on test because in GitHub the indentation is more than required and the image appears as code instead.
![image](https://github.com/user-attachments/assets/8944571c-15cc-4b48-8da9-cfdd0ddd3111)